Library/Formula/polo.rb

```diff
@@ -0,0 +1,22 @@
+class Polo < Formula
+  desc "Static blog generator compatible with Jekyll & Pelican content"
+  homepage "https://github.com/agonzalezro/polo"
+  url "https://github.com/agonzalezro/polo/archive/v1.0.tar.gz"
+  sha256 "efe7e798212b05b76ab63f32603bc15eedd9eadec3bfbfc95e36c64ba13f00dc"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = bin
+    ENV["GOPATH"] = buildpath
+    ENV["GOHOME"] = buildpath
+
+    system "go", "get", "github.com/constabulary/gb/..."
```

Comment:
Please use the `go_resource` method instead of `go get`. Example [here](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/consul-template.rb#L131-L144).